src/models.py

```diff
@@ -35,3 +41,142 @@ def __init__(self, hopeful):
 
     def __repr__(self):
         return "<Hopeful %i>" % self.hash_code
+
+class UserSchool(db.Model):
+    __tablename__ = 'userschool'
```

Comment:
Modified for simple tables. Left the names unchanged for many-to-many table relations